m4/ax_fresh_compiler.m4

```diff
@@ -47,7 +47,6 @@ AC_DEFUN([AX_FRESH_COMPILER],
                 auto a = sizeof(std::max_align_t);
                 auto b = std::get_new_handler();
                 auto c = std::get_terminate();
-                auto d = std::get_unexpected();
```

Comment:
did you just remove this or picked up a more recent version of this file?

Reply:
We wrote this file! But std::get_unexpected was removed in C++17.